.pre-commit-config.yaml

```diff
@@ -97,18 +97,39 @@ repos:
           - types-tqdm
           - types-flask
 
+  # Check for spelling
+  - repo: https://github.com/crate-ci/typos
+    rev: v1.23.6
+    hooks:
+      - id: typos
+        exclude: .html
+
   # Catch common capitalization mistakes
   - repo: local
     hooks:
       - id: disallow-caps
         name: Disallow improper capitalization
         language: pygrep
         entry: PyBind|Numpy|Cmake|CCache|Github|PyTest|Mqt|Tum
-        exclude: .pre-commit-config.yaml
+        exclude: \.pre-commit-config\.yaml|\.joblib$
 
   # Check best practices for scientific Python code
   - repo: https://github.com/scientific-python/cookie
     rev: 2024.04.23
     hooks:
       - id: sp-repo-review
         additional_dependencies: ["repo-review[cli]"]
+
+  # Check JSON schemata
+  - repo: https://github.com/python-jsonschema/check-jsonschema
+    rev: 0.29.1
+    hooks:
+      - id: check-dependabot
+      - id: check-github-workflows
+      - id: check-readthedocs
+
+  # Check the pyproject.toml file
+  - repo: https://github.com/henryiii/validate-pyproject-schema-store
+    rev: 2024.08.08
+    hooks:
+      - id: validate-pyproject
```

Comment:
Might be worth running `pre-commit autoupdate` once because I think that new versions of `cookie` and the `validate-pyproject-schema-store` came out in the last couple of days.
I believe these also make the `cookie` ignores in the `pyproject.toml` obsolete.

Reply:
Done and removed the ignores.